### ISSUE TITLE
Bower install fails

### ISSUE THREAD

**Issue by mikkotikkanen** [cloudfour/hideShowPassword#44]
At least at Windows machines (haven't tested with others OS, but with multiple Windows machines), the bower install fails as bower.json seems to be a symbolic link with "package.json" text within it. The symbolic link seems to break down for some reason and since the "package.json" isn't exactly valid JSON, the install aborts.

```
bower hideshowpassword#*                               EMALFORMED Failed to read
<temp path here>\bower.json

Additional error details:
Unexpected token p
```

**Comment by mikkotikkanen** [comment]
Also, 2.0.3 seems to install correctly

**Comment by tylersticka** [comment]
Thanks, @mikkotikkanen. The `package.json` file is valid JSON, but it may not be a valid `bower.json`.

**Comment by tylersticka** [comment]
Hmm, seems to work fine on Mac:

```
$ bower install hideshowpassword
bower cached        https://github.com/cloudfour/hideShowPassword.git#2.0.10
bower validate      2.0.10 against https://github.com/cloudfour/hideShowPassword.git#*
bower cached        https://github.com/jquery/jquery-dist.git#2.2.3
bower validate      2.2.3 against https://github.com/jquery/jquery-dist.git#>=1.11
bower install       hideshowpassword#2.0.10
bower install       jquery#2.2.3

hideshowpassword#2.0.10 bower_components/hideshowpassword
└── jquery#2.2.3

jquery#2.2.3 bower_components/jquery
```

Will attempt to troubleshoot on Windows.

**Comment by tylersticka** [comment]
Verified on Windows.

The issue seems to be with the symlink. See: yeoman/yeoman#751

Solution may be to just remove `bower.json` file entirely. More research needed, as our team no longer uses Bower.

**Comment by tylersticka** [comment]
@mikkotikkanen Thanks for reporting the issue! Should be fixed in `v2.0.11` (just merged).
